yelp_package/dockerfiles/trusty/Dockerfile

```diff
@@ -14,12 +14,18 @@
 
 FROM ubuntu:trusty
 
+RUN echo "deb http://repos.mesosphere.com/ubuntu trusty main" > /etc/apt/sources.list.d/mesosphere.list
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv 81026D0004C44CF7EF55ADF8DF7D54CBE56151BF
+
 RUN apt-get update && apt-get -y install dpkg-dev python-tox python-setuptools \
   python-dev debhelper python-yaml libyaml-dev python-pytest pyflakes \
-  git help2man zsh wget
+  git help2man zsh wget mesos=0.28.2-2.0.27.ubuntu1404
 
 RUN cd `mktemp -d` && wget http://mirrors.kernel.org/ubuntu/pool/universe/d/dh-virtualenv/dh-virtualenv_0.11-1_all.deb && dpkg -i dh-virtualenv_0.11-1_all.deb && apt-get -f install
 
+RUN apt-get install -yq zip
+RUN cd /usr/lib/python2.7/site-packages && zip -r /root/mesos.native-0.28.2-py27-none-any.whl mesos/native mesos.native-0.28.2.dist-info
```

Comment:
Why do you need this? This should be handled by the normal virtualenv constructs right?